multiapi-engine/src/test/resources/asyncapigenerator/testCustomValidators/customvalidator/SizeValidator.java

```diff
@@ -0,0 +1,36 @@
+package com.sngular.scsplugin.customvalidator.model.event.customvalidator;
+
+import java.util.Objects;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class SizeValidator implements ConstraintValidator<Size, String> {
+
+    private int min;
+    private int max;
+
+    @Override
+    public void initialize(Size constraintAnnotation) {
+        ConstraintValidator.super.initialize(constraintAnnotation);
+        this.min = constraintAnnotation.min();
+        this.max = constraintAnnotation.max();
+    }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (Objects.isNull(value))
```

Comment:
``` return Objects.isNull(value) || ((max==0 || value.length() <= max) && (min!=0 || value.length() >= min));```

Reply:
The logic needed some changes, but this is already done